### PR TITLE
Fix DTLS related crash (issue #861)

### PR DESCRIPTION
**Pull request by ibc**
It may fix issue #861, which was potentially introduced by PR #815.

WIP: no idea what's going on here.

### CODE REVIEW

**Comment by ibc** [comment]
Asking for @fippo's and @ggarber thoughts.

File: worker/src/RTC/DtlsTransport.cpp [hunk 1]
```diff
@@ -1432,19 +1431,18 @@ namespace RTC
 		{
 			MS_WARN_TAG(dtls, "DTLSv1_handle_timeout() failed");
 
-			Reset();
+			// NOTE: Don't do anything here (such as calling Reset() because the error
```
Comment (ggarber) [inline worker/src/RTC/DtlsTransport.cpp]: who is going to call CheckStatus in this case?

**Comment by fippo** [comment]
would it be possible for the timer to survive the object? That is the only thing I can think of...

File: worker/src/RTC/DtlsTransport.cpp (hunk 1 above)
Comment (ibc) [reply worker/src/RTC/DtlsTransport.cpp]: Right, it shouldn't...

**Comment by ibc** [comment]
> would it be possible for the timer to survive the object? That is the only thing I can think of...

Nope, of course the timer is properly destroyed in the `DtlsTransport` destructor.

File: worker/src/RTC/DtlsTransport.cpp (hunk 1 above)
Comment (ibc) [reply worker/src/RTC/DtlsTransport.cpp]: Reverted.

**Comment by ibc** [comment]
Released in 3.10.4

**Pull request merged**